geopandas/geodataframe.py

```diff
@@ -459,6 +459,17 @@ def __getitem__(self, key):
             result.__class__ = DataFrame
         return result
 
+
+    def explode(self):
```

Comment:
I would put this a bit below in the file, eg just after the definition of `dissolve`

Reply:
Fixed! Wasn't sure where it beloneged :)